monument/test/results.toml

```diff
@@ -737,6 +738,122 @@ len    Y   C |  music  | avg score | calling
+1056 |  304.00 :   12     0     86 (  38f   48b)      0 |  0.283523 | sHsH
+-----|--------------------------------------------------|-----------|-----------
+ len |  music     65s   65s       4-bell runs       *TE | avg score | calling
+'''
+"test/cases/complib-music/minor.toml" = '''
+len |  music     *65 | avg score | calling
+----|----------------|-----------|-----------
+144 |   30.00 :    0 |  0.183333 | IW
+120 |   30.00 :    0 |  0.250000 | 
+144 |   43.00 :    0 |  0.273611 | HB
+144 |   44.00 :    0 |  0.280556 | BH
+ 72 |   46.00 :    0 |  0.563889 | HHH
+216 |   49.00 :    0 |  0.176852 | IBWWHH
+216 |   51.00 :    0 |  0.186111 | HIBWWH
+216 |   59.00 :    0 |  0.223148 | HHIBWW
+----|----------------|-----------|-----------
+len |  music     *65 | avg score | calling
+'''
+"test/cases/complib-music/royal.toml" = '''
+len |  music     56s   56s       4-bell runs       *09 | avg score | calling
+----|--------------------------------------------------|-----------|-----------
+602 |   70.00 :    3     0     28 (  11f   17b)      0 |  0.088206 | sMxBsMxBsW>
+602 |   72.00 :    3     0     32 (  13f   19b)      0 |  0.083887 | sMsWxBsMxBsMsW>
+602 |   72.00 :    3     0     32 (  13f   19b)      0 |  0.085548 | sMsWxBMxBMsW>
+602 |   88.00 :    3     0     38 (  15f   23b)      0 |  0.119767 | MxBMxBsW>
+562 |   99.00 :    3     0     34 (  13f   21b)      0 |  0.168861 | WsW>
+560 |  108.00 :    6     0     41 (  17f   24b)      0 |  0.169286 | sMWxBMsW
+360 |  118.00 :    6     0     37 (  15f   22b)      0 |  0.327778 | 
+560 |  124.00 :    5     1     52 (  20f   32b)      0 |  0.203571 | xBxH
+560 |  126.00 :    6     0     51 (  21f   30b)      0 |  0.209643 | MxBW
+----|--------------------------------------------------|-----------|-----------
+len |  music     56s   56s       4-bell runs       *09 | avg score | calling
+'''
+"test/cases/complib-music/triples.toml" = '''
+len |  music     567s   657s   CRUs    4-bell runs   46s | avg score | calling
+----|----------------------------------------------------|-----------|-----------
+56 |   38.00 :    2      2       4    6 ( 3f  3b)     5 |  0.532143 | sWsFMH
+56 |   38.00 :    2      2       4    6 ( 3f  3b)     5 |  0.550000 | WFMH
+56 |   40.00 :    2      2       4    6 ( 3f  3b)     6 |  0.567857 | WsFsMH
+56 |   40.00 :    2      2       4    6 ( 3f  3b)     6 |  0.567857 | sWFsMH
+56 |   41.00 :    2      2       4    8 ( 4f  4b)     5 |  0.567857 | sWsFsMsH
+56 |   41.00 :    2      2       4    8 ( 4f  4b)     5 |  0.585714 | WFsMsH
+56 |   43.00 :    2      2       4    8 ( 4f  4b)     6 |  0.621429 | WsFMsH
+56 |   43.00 :    2      2       4    8 ( 4f  4b)     6 |  0.621429 | sWFMsH
+84 |   76.00 :    4      4       8   14 ( 7f  7b)     9 |  0.904762 | 
+----|----------------------------------------------------|-----------|-----------
+len |  music     567s   657s   CRUs    4-bell runs   46s | avg score | calling
```

Comment:
Oh exciting, you've managed to break the column alignment by making a composition with a 2-digit maximum length... this was an oversight on my part and I've quickly fixed it in #309.

I think we merge this PR first, then I'll rebase #309 onto this one and change this test in that PR.  So, no action required from you here :)